.github/workflows/buildtest.yml

```diff
@@ -18,10 +18,9 @@ env:
   # github runners currently have two cores
   NR_JOBS: "2"
 
-  CARL_PARSER_GIT_URL: "https://github.com/ths-rwth/carl-parser"
-  CARL_PARSER_BRANCH: "master14"
   CARL_GIT_URL: "https://github.com/ths-rwth/carl"
```

Comment:
Shouldnt these also be changed?

Reply:
This will be changed with https://github.com/moves-rwth/pycarl/pull/20

Reply:
`CARL_GIT_URL` will be changed with https://github.com/moves-rwth/pycarl/pull/20